### ISSUE TITLE
[Bug]: tae w-w conflict when run tpcc benchmark

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#5581]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):0.6.0
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

Lots of w-w conflict when run tpcc with 1 warehouse 10 terminals

2022/10/12 18:13:03.574674 +0800 ERROR frontend/util.go:454 query trace status {"connection_id": 1010, "statement": "update bmsql_district set d_next_o_id = bmsql_district.d_next_o_id + 1 where bmsql_district.d_w_id = ? and bmsql_district.d_id = ?", "status": "fail", "error": "w-w conflict"}
github.com/matrixorigin/matrixone/pkg/frontend.logStatementStringStatus
        /data1/sudong/mo-regression/matrixone/pkg/frontend/util.go:454
github.com/matrixorigin/matrixone/pkg/frontend.logStatementStatus
        /data1/sudong/mo-regression/matrixone/pkg/frontend/util.go:446
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery
        /data1/sudong/mo-regression/matrixone/pkg/frontend/mysql_cmd_executor.go:2488
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest
        /data1/sudong/mo-regression/matrixone/pkg/frontend/mysql_cmd_executor.go:2611
github.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop
        /data1/sudong/mo-regression/matrixone/pkg/frontend/routine.go:125
2022/10/12 18:13:03.574723 +0800 ERROR frontend/util.go:454 query trace status {"connection_id": 1011, "statement": "update bmsql_district set d_next_o_id = bmsql_district.d_next_o_id + 1 where bmsql_district.d_w_id = ? and bmsql_district.d_id = ?", "status": "fail", "error": "w-w conflict"}
github.com/matrixorigin/matrixone/pkg/frontend.logStatementStringStatus
        /data1/sudong/mo-regression/matrixone/pkg/frontend/util.go:454
github.com/matrixorigin/matrixone/pkg/frontend.logStatementStatus
        /data1/sudong/mo-regression/matrixone/pkg/frontend/util.go:446
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery
        /data1/sudong/mo-regression/matrixone/pkg/frontend/mysql_cmd_executor.go:2488
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest
        /data1/sudong/mo-regression/matrixone/pkg/frontend/mysql_cmd_executor.go:2611
github.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop
        /data1/sudong/mo-regression/matrixone/pkg/frontend/routine.go:125
2022/10/12 18:13:03.574874 +0800 ERROR frontend/mysql_cmd_executor.go:2479 w-w conflict
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery
        /data1/sudong/mo-regression/matrixone/pkg/frontend/mysql_cmd_executor.go:2479
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest
        /data1/sudong/mo-regression/matrixone/pkg/frontend/mysql_cmd_executor.go:2611
github.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop

ithub.com/matrixorigin/matrixone/pkg/frontend.logStatementStringStatus
        /data1/sudong/mo-regression/matrixone/pkg/frontend/util.go:454
github.com/matrixorigin/matrixone/pkg/frontend.logStatementStatus
        /data1/sudong/mo-regression/matrixone/pkg/frontend/util.go:446
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery
        /data1/sudong/mo-regression/matrixone/pkg/frontend/mysql_cmd_executor.go:2488
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest
        /data1/sudong/mo-regression/matrixone/pkg/frontend/mysql_cmd_executor.go:2611
github.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop
        /data1/sudong/mo-regression/matrixone/pkg/frontend/routine.go:125
2022/10/12 18:13:03.574874 +0800 ERROR frontend/mysql_cmd_executor.go:2479 w-w conflict
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery
        /data1/sudong/mo-regression/matrixone/pkg/frontend/mysql_cmd_executor.go:2479
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest
        /data1/sudong/mo-regression/matrixone/pkg/frontend/mysql_cmd_executor.go:2611
github.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop
        /data1/sudong/mo-regression/matrixone/pkg/frontend/routine.go:125
2022/10/12 18:13:03.574993 +0800 ERROR frontend/util.go:454 query trace status {"connection_id": 1005, "statement": "update bmsql_district set d_next_o_id = bmsql_district.d_next_o_id + 1 where bmsql_district.d_w_id = ? and bmsql_district.d_id = ?", "status": "fail", "error": "w-w conflict"}
github.com/matrixorigin/matrixone/pkg/frontend.logStatementStringStatus
        /data1/sudong/mo-regression/matrixone/pkg/frontend/util.go:454
github.com/matrixorigin/matrixone/pkg/frontend.logStatementStatus
        /data1/sudong/mo-regression/matrixone/pkg/frontend/util.go:446
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery
        /data1/sudong/mo-regression/matrixone/pkg/frontend/mysql_cmd_executor.go:2488
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest

**Comment by tianyahui-python** [comment]
@aressu1985 pls update the performance on latest commit

**Comment by aressu1985** [comment]
fixed: 249f56400184a0828afdc615c8b5d75e8c57cecc
